lib/util.js

```diff
@@ -128,7 +128,7 @@ module.exports = {
       restorer
     }
     state.value = state.value.replace(what, (c) => {
-      if (exclude && c.match(exclude)) {
+      if (exclude && exclude.test(c)) {
```

Comment:
This needs careful review; if the global flag is used, then this is not the same as before.

Reply:
Yes, global flag is used.